### ISSUE TITLE
What is an EpisodeAdded?

### ISSUE THREAD

**Issue by drozzy** [deepmind/acme#38]
Having read the [docs](https://github.com/deepmind/acme/blob/master/docs/components.md) and the code for the [episode adder](https://github.com/deepmind/acme/blob/master/acme/adders/reverb/episode.py#L51) I still don't quite understand it.

Is it simply the simplest added? E.g. adding every transition to the buffer? Or is it doing something else?

Thanks!

**Comment by Bihaqo** [comment]
It's an adder which adds entire episodes into replay. For this to make sense your agent has to be recurrent.
An alternative is e.g. adding transitions (SARSA tuples) into replay: NStepTransitionAdder will slice your episode into SARSA chunks and upload each to replay independently.

**Comment by aslanides** [comment]
Yes, this is exactly right. In contrast to the transition adder, which turns things into (possibly n-step) transitions, or the sequence adder, which slices up the episode into (possibly strided, possibly overlapping) sequences of fixed length, this just creates one sequence of arbitrary length, depending on the episode length.

**Comment by aslanides** [comment]
We could improve the documentation of this.

**Comment by drozzy** [comment]
So I guess replay buffer doesn't only store "transitions", but "sequences", and a transition is a special case of such a sequence? 
<img width="249" alt="Screen Shot 2020-07-02 at 2 08 01 AM" src="https://user-images.githubusercontent.com/140710/86322377-dce55a00-bc08-11ea-9e5b-7fe3c00564d3.png">

**Comment by Bihaqo** [comment]
Not exactly. When you work with SARSA tuples it indeed works like this. When working with sequences, they are stored as a tuple `(observations, actions, rewards, discounts)`, where each tensor has shape `[batch, sequence_length, ...]`.
So e.g. `a_1 = actions[:, 1]`

**Comment by drozzy** [comment]
Oh got it.

Perhaps adding dimensions to the docs is a good idea. Looking at that picture above it is not that clear.
But after your explanation it is immediately clear.
